app/views/local_transaction/unavailable_service.html.erb

```diff
@@ -1,34 +1,42 @@
 <%= render layout: 'shared/base_page', locals: {
-  title: @service.title,
+  title: t('formats.local_transaction.service_not_available', country_name: @country_name),
   publication: @publication,
   edition: @edition,
 } do %>
   <div class="interaction">
     <p class="govuk-body">
-      We've matched the postcode to <span class="local-authority"><%= @local_authority.name %></span>.
+      <%= t('formats.local_transaction.matched_postcode_html', local_authority: @local_authority.name) %>
     </p>
 
-  <% if @service.body.present? %>
-    <p class="govuk-body"><%= @service.body %></p>
-  <% end %>
-
-    <p id="get-started" class="get-started group">
-      <%= render "govuk_publishing_components/components/button", {
-        text: @service.button_text,
-        rel: "external",
-        margin_bottom: true,
-        href: @service.button_link,
-      } %>
+    <p class="govuk-body">
+      <%= t('formats.local_transaction.service_not_available', country_name: @country_name) %>.
     </p>
-  </div>
 
-  <% if @publication.more_information.present? %>
-    <section class="more">
-      <div class="more">
-        <%= render "govuk_publishing_components/components/govspeak", {} do %>
-          <%= raw @publication.more_information %>
-        <% end %>
+    <% if @local_authority.url.present? %>
+      <p class="govuk-body">
+        <%= t('formats.local_transaction.find_other_services') %>
+      </p>
+      <div class="local-authority-result"
+           data-module="auto-track-event"
+           data-track-category="postcodeSearch:local_transaction"
+           data-track-action="postcodeResultShown:devolvedAdministrationUnavailable"
+           data-track-label=<%= @country_name %>>
+        <p id="get-started" class="get-started group">
+          <%= render "govuk_publishing_components/components/button", {
+            text: t('formats.local_transaction.local_authority_website', local_authority_name: @local_authority.name),
+            rel: "external",
+            margin_bottom: true,
+            href: @local_authority.url,
+          } %>
+        </p>
       </div>
-    </section>
-  <% end %>
+    <% else %>
+      <div class="local-authority-result"
+      data-module="auto-track-event"
+      data-track-category="userAlerts:local_transaction"
+      data-track-action="postcodeResultShown:laMatchNoLinkNoAuthorityUrl">
+        <p class="govuk-body"><%= t('formats.local_transaction.no_local_authority_url_html') %></p>
+      </div>
```

Comment:
Shall we put this in a partial so that it doesn't need to be duplicated between this and results? That seems to offer a slightly better chance that this stays up to date. Could probably call it `_no_local_authority_url.html.erb` to match the translation key.

Reply:
Good idea! Have put in a partial.